api/views.py

```diff
@@ -178,6 +181,35 @@ def get_queryset(self):
         return super().get_queryset().select_related('upload_endpoint')
 
 
+class MediaItemEmbedViewInspector(inspectors.ViewInspector):
```

Comment:
seem to be missing imports for inspectors/openapi - if this has been addressed in a subsequent commit then ignore